src/modules/app/StoreActionType.js

```diff
@@ -109,7 +110,7 @@ export type EventsActionType =
 
 export type SwitchContentLanguageActionType = {|
   type: 'SWITCH_CONTENT_LANGUAGE', +params: {|
-    +newLanguage: string, +city: string
+    +newLanguage: string, +city: string, +t: TFunction
```

Comment:
I'd like to avoid having functions in actions. Is there another way to give the sagas the translate function? For example by setting up the i18n before the redux store; That way we could also initialize the content language in the store directly with the information provided by i18n. Also you would have to touch a lot less files

Reply:
Since the alert will be replaced by the error component in https://issues.integreat-app.de/browse/NATIVE-359  ,hence the TFunction will not be required anymore anyway. Therefore we agreed on keeping this at it is.